.github/workflows/phpcs.yml

```diff
@@ -0,0 +1,20 @@
+name: Coding Standards
+
+on: ['workflow_call']
+
+jobs:
+  phpcs:
+    name: phpcs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Configure PHP environment
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.0'
+      - uses: ramsey/composer-install@v2
```

Comment:
This needs to also get the .env secrets, or we will never get past the `composer install`.